tests/test_fmi3.py

```diff
@@ -581,6 +580,28 @@ def test_get_input_list(self):
 
         assert expected == list(inputs.keys())
 
+    def test_get_int32_and_64(self):
+        """ Test get int32 and int64. """
+        fmu_path = FMI3_REF_FMU_PATH / "Feedthrough.fmu"
+        fmu = load_fmu(fmu_path)
+        int64_vals = fmu.get_int64([23, 24])
+        int32_vals = fmu.get_int32([19, 20])
+
+        np.testing.assert_equal(np.array([0, 0]), int64_vals)
```

Comment:
A thought; maybe we want to implement the setters as well and properly test:
1. Getting/setting works e.g., for int64s for values that would not fit into an int32
2. Connected to that also the practical limitations of storing results for these?

Reply:
Ill create a separate issue for this.